projects/mercury/src/metadata/common/LinkedDataLink.js

```diff
@@ -1,28 +1,61 @@
 import React, {useContext} from 'react';
-import {Link as RouterLink} from "react-router-dom";
 import * as PropTypes from "prop-types";
-import {Link} from '@mui/material';
-import {METADATA_PATH} from "../../constants";
+import {Box, Modal, Tooltip} from '@mui/material';
+import CloseIcon from '@mui/icons-material/Close';
+import LinkedDataEntityPage from "./LinkedDataEntityPage";
 import UserContext from '../../users/UserContext';
-
 /**
  * Renders a link to the metadata editor.
  *
  * @param props
  * @constructor
  */
+
+const styleModalDialog = {
+    position: 'absolute',
```

Comment:
I see that we use '.style.js' files to define styles. What do you think if we move it to a dedicated one?